applications/luci-app-openwisp/Makefile

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Nicholas Smith (nicholas@nbembedded.com)
+# This is free software, licensed under the GNU General Public License v2.
+
+include $(TOPDIR)/rules.mk
+
+PKG_LICENSE:=GPL-2.0+
```

Comment:
This one is old SPDX License Identifier.

Reply:
fixed